docs/sce_file_contents.md

```diff
@@ -124,44 +126,46 @@ The following command can be used to access the UMAP results:
 reducedDim(sce,"UMAP")
 ```
 
-## Additional SingleCellExperiment components for CITE-seq libraries
+## Additional SingleCellExperiment components for ADT (CITE-seq) libraries
```

Comment:
The whole library is CITE-seq, so we can call it that first and put the ADT in the parentheses
```suggestion
## Additional SingleCellExperiment components for CITE-seq libraries (with ADT tags)
```